src/utils/Processing.ts

```diff
@@ -564,6 +567,42 @@ export function writeFHIRResources(
   writeResources(outPackage.instances.filter(i => i._instanceMeta.usage !== 'Inline'));
 
   logger.info(`Exported ${count} FHIR resources as JSON.`);
+  return skippedResources;
+}
+
+export function writeFSHIndex(
+  outDir: string,
+  outPackage: Package,
+  inputDir: string,
+  skippedResources: string[] = []
+) {
+  const index: string[][] = [];
+  const otherIndex: any[] = [];
```

Comment:
Is it possible to name these a little more descriptively? Maybe `textIndex` and `jsonIndex`? I'm not sure but I had to figure out for a second what the difference was.